### PR TITLE
16: Signup Module

### PR DESCRIPTION
**Pull request by SakshiShreya**

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import dotenv from "dotenv";
+import express from "express";
+import { authenticateByGoogle, signupByEmail } from "../controllers/users.js";
+
+dotenv.config({ path: "./config.env" });
```
Comment (Soumya-Shreya) [inline routes/users.js:5]: Why is this needed?

Comment (SakshiShreya) [reply routes/users.js:5]: This allows to use environment variables. I am moving this line to controller to avoid confusion.

**Review by Van-sh — APPROVED** [review]

**Review by Soumya-Shreya — APPROVED** [review]
